src/utils/helpers.ts

```diff
@@ -54,6 +62,35 @@ const getPlatformInfo = (platformName: string) => {
   else return { shortName: '', icon: {} };
 };
 
+const getDocsURL = (suite: string, framework_id: Framework['id']) => {
+  const framework = frameworkMap[framework_id];
+  const supportedFramework =
+    supportedPerfdocsFrameworks[framework as SupportedPerfdocsFramework];
+  const urlReadySuite = suite.replace(/:|\s|\.|_/g, '-').toLowerCase();
+
+  let docsURL = '';
+  let isLinkSupported = true;
+
+  const isDevToolsFramework = framework_id === 12;
+
+  if (isDevToolsFramework && supportedFramework) {
+    docsURL = `${baseDocsURL}/devtools/tests/${supportedFramework}.html#${urlReadySuite}`;
+    if (
+      suite === removedOldTestDevTools ||
+      nonDocumentedTestsDevTools.includes(suite)
+    ) {
+      isLinkSupported = false;
+      docsURL = '';
+    }
```

Comment:
You could move this logic into a new helper function (eg `createDevtoolsDocsUrl`)
```suggestion
    docsURL = createDevtoolsDocsUrl(...)
```